.php_cs

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return Symfony\CS\Config\Config::create()
+    ->fixers([
+        '-phpdoc_short_description',
```

Comment:
@Soullivaneuh @stloyd 
Remove all -[fixers] and leave the default cs-fixer Symfony config?